_sass/category.scss

```diff
@@ -40,31 +39,18 @@
   }
 }
 
-.category--discover {
-  background-color: $color-blue;
-}
-
-.category--make {
-  background-color: $color-gold;
-}
-
-.category--validate {
-background-color: $color-red;
-}
-
-.category--decide {
-background-color: $color-green;
-}
+.category--discover { background-color: $color-blue; }
```

Comment:
I'd recommend following the css formatting guidelines here https://frontend.18f.gov/#css-formatting